src/package.json

```diff
@@ -19,6 +19,9 @@
     "watch": "node ./tools/generate/chapter_watcher",
     "ebooks": "node ./tools/generate/generate_ebook_pdfs",
     "deploy": "./tools/scripts/deploy.sh",
+    "lint": "run-script-os",
+    "lint:darwin:linux": "docker container run -it --rm -v $PWD/..:/app -w /app/src --entrypoint=./tools/scripts/run_linter_locally.sh github/super-linter",
```

Comment:
It is generally a good idea to place double quotes around `"$PWD"` when used in commands like these to avoid unintentional argument split in case any parent directory name has spaces in it.

Reply:
Done